.github/workflows/jacoco_check.yml

```diff
@@ -51,24 +50,38 @@ jobs:
           java-version: "adopt@1.8"
       - name: Build and run tests
         run: sbt ++${{matrix.scala}} jacoco -DSPARK_VERSION=${{matrix.spark}}
-      - name: Add coverage to PR
+      - name: Add coverage of 'cobol-parser' to PR
+        id: jacocoParser
+        uses: madrapps/jacoco-report@v1.3
+        with:
+          paths: >
+            ${{ github.workspace }}/cobol-parser/target/scala-${{ matrix.scalaShort }}/jacoco/report/jacoco.xml
+          token: ${{ secrets.GITHUB_TOKEN }}
+          min-coverage-overall: ${{ matrix.overall }}
+          min-coverage-changed-files: ${{ matrix.cobolParserOverride }}
+          title: JaCoCo code coverage report - 'cobol-parser'
+          update-comment: true
+      - name: Get the Coverage info
```

Comment:
I am missing Fail steps for cobolParser. See proposal:
`
      - name: Fail PR if changed files coverage is less than ${{ matrix.cobolParserOverride }}%
        if: ${{ steps.jacocoParser.outputs.coverage-changed-files < matrix.cobolParserOverride }}
        uses: actions/github-script@v6
        with:
          script: |
            core.setFailed('Changed files coverage is less than ${{ matrix.cobolParserOverride }}%!')
`

Reply:
Sorry, i do not understand. What is your suggestion?

Please, use triple back quotes (```) to surround code blocks. Or better yet, you can use the suggestion button:
![Screenshot 2023-04-24 at 9 33 45](https://user-images.githubusercontent.com/4082463/233929229-7111fdf3-cdb5-4ccb-a026-7457db5d3656.png)

Reply:
See new Change proposal comment, please.